c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -136,6 +136,12 @@ Version 1.6
 13. Remove your Apache configuration file, probably in
     `/var/www/vhost/{{project}}/conf`. Now it will be automatically created.
 
+14. Notes on database changes:
+    * We add some tables and do some changes for GeoMapFish 2.0
+    * We move the user table in the static schema than all the data that
```

Comment:
Table ``user`` has been moved to schema ``main_static`` in order to store all data modified by the web application in the same schema.